sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -351,15 +351,27 @@ def on_response(self, request, response):
         except Exception as err:  # pylint: disable=broad-except
             _LOGGER.debug("Failed to log response: %s", repr(err))
 
+class _HiddenClassProperties(type):
+    # Backward compatible for DEFAULT_HEADERS_WHITELIST but no documentation needed
+    # https://github.com/Azure/azure-sdk-for-python/issues/26331
 
-class HttpLoggingPolicy(SansIOHTTPPolicy):
+    @property
+    def DEFAULT_HEADERS_WHITELIST(cls):
+        return cls.DEFAULT_HEADERS_ALLOWLIST
+
+    @DEFAULT_HEADERS_WHITELIST.setter
+    def DEFAULT_HEADERS_WHITELIST(self, value):
+        self.DEFAULT_HEADERS_ALLOWLIST = value
```

Comment:
This only works to set on an instance(`self`) not the class type (`cls`)?

Reply:
Ignore that - I see in your tests you're testing setting to the class type - so looks like it's good.

Reply:
Ha yeah, used `self` by habit, should have called it `cls`